src/Device/style.css

```diff
@@ -0,0 +1,307 @@
+
+.ola_device *,
+.ola_device ::after,
+.ola_device ::before,
+.ola_device::after,
+.ola_device::before {
+  box-sizing: border-box;
+  display: block;
+}
+.ola_device {
+  --scale: 0.5;
+  position: relative;
+  z-index: 1;
+}
+
+.ola_device .ola_device-frame {
+    z-index: 1; 
+  }
+  .ola_device .ola_device-content {
+    background-color: #fff;
+    background-position: center center;
+    background-size: cover;
+    object-fit: cover;
```

Comment:
Good one the `object-fit`. I'd include also `object-position: center top` so the top of the image is always visible.

Reply:
totally right with u! added 💪🏻